installation/install.sh

```
@@ -4684,6 +4684,10 @@ setup_install_param
 
 #Delete old monitoring data if NEED_DROP_DB
 if [ -n "$NEED_DROP_DB" ]; then
+  # Stop mn process before dropping DB to avoid DB operation failures
+  zstack-ctl stop_node 2>/dev/null || true
+  ZSTACK_PID=`ps aux | grep 'appName=zstack' | grep -v 'grep' | awk '{ print $2 }'`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)